docs/metrics.md

```diff
@@ -28,20 +29,39 @@ This is the metrics collection plan for Lockbox's alpha release. It is more of a
 
 Best viewed in something that can render Markdown.
 
-## TL;DR Metrics Priorities for Alpha
+## TL;DR Metrics Implemented for Alpha Thus Far
+
+These are the metrics that have been implemented in the pull request that included the edits to this doc.
+
+All events are under the **category: lockbox**. The `extra` field always contains `fxaid` where possible (i.e. after FxA auth). In the future also the `itemid` where relevant.
+
+1. `startup` fires when the webextension is loaded. **method**:click **objects**: addon, webextension.
```

Comment:
I have a feeling this list isn't going to render the numbers correctly in Markdown; it might look like:

```
1. `startup`....
1. `iconClick`...
```

That's because the actual number used in an ordered list in Markdown isn't consulted as I recall. It'd be good to check this to be sure...

Reply:
This works as expected.  Markdown largely ignores the numerical value; instead if it sees the pattern `<number>.` at the start of a line, it treats it as an ordered list (`\<ol>`).

Reply:
Ok, cool. I just wasn't sure if the extra newlines were actually creating multiple `<ol>`s.